combat-trainer.lic

```diff
@@ -313,7 +313,7 @@ class SetupProcess
       game_state.prepare_summoned_weapon(last_summoned)
     else
       bput('aim stop', "But you're not aiming", 'You stop concentrating', 'You are already') if game_state.aimed_skill?
-      @equipment_manager.wield_weapon(game_state.weapon_name, game_state.weapon_skill)
+      game_state.wield_weapon(game_state.weapon_skill)
```

Comment:
This is calling a function in game_state and passing it a value it already has. Each of the _Process_ callers of the function pass the same first arg and no second arg. What do you think about adding a new function `wield_weapon_for_current_skill` (or whatever name you prefer) that takes no args?